src/main/kotlin/com/personio/synthetics/builder/AssertionsBuilder.kt

```diff
@@ -0,0 +1,98 @@
+package com.personio.synthetics.builder
+
+import com.datadog.api.client.v1.model.SyntheticsAssertion
+import com.datadog.api.client.v1.model.SyntheticsAssertionJSONPathOperator
+import com.datadog.api.client.v1.model.SyntheticsAssertionJSONPathTarget
+import com.datadog.api.client.v1.model.SyntheticsAssertionJSONPathTargetTarget
+import com.datadog.api.client.v1.model.SyntheticsAssertionOperator
+import com.datadog.api.client.v1.model.SyntheticsAssertionTarget
+import com.datadog.api.client.v1.model.SyntheticsAssertionType
+
+class AssertionsBuilder {
+    private val assertions = mutableListOf<SyntheticsAssertion>()
+
+    fun build(): List<SyntheticsAssertion> {
+        return assertions
+    }
+
+    /**
+     * Asserts the response status code
+     * @param code Status code
+     */
+    fun statusCode(code: Int) {
+        target(
+            SyntheticsAssertionType.STATUS_CODE,
+            SyntheticsAssertionOperator.IS,
+            code
+        )
+    }
+
+    /**
+     * Asserts that response header contains given value
+     * @param name Header name
+     * @param value Value to look for
+     */
+    fun headerContains(name: String, value: String) {
+        assertions.add(
+            SyntheticsAssertion(
+                SyntheticsAssertionTarget()
+                    .property(name)
+                    .operator(SyntheticsAssertionOperator.CONTAINS)
+                    .type(SyntheticsAssertionType.HEADER)
+                    .target(value)
+            )
+        )
+    }
+
+    /**
+     * Asserts that the value at the json path in the response body  contains given value
```

Comment:
nit: Looks like there are 2 spaces between `body  contains`. Would be nice to remove one.

Also, perhaps, it's better to use `JSON` word format like we do for JS/Javascript.

And, given value -> the given value